CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Shift + Backspace` now sends `^?` instead of `^H`
 - Default color scheme is now `Tomorrow Night` with the bright colors of `Tomorrow Night Bright`
 - Set IUTF8 termios flag for improved UTF8 input support
+- Dragging multiple files into terminal now adds a space after each file
```

Comment:
This is not entirely related to multiple files, just say that space is added after dragging file, I guess.

Reply:
You got. I’ve changed it to reflect that it is after each file name. Not just multiple.